### ISSUE TITLE
Feature request: make requestFormDataBody support requestBody types

### ISSUE THREAD

**Issue by zecar** [ad-on-is/adonis-autoswagger#73]
I have an endpoint for a model that requires another field for file upload

It would be nice to not be forced to write the whole model as json to make it work for formData. It should work like

`<Model>.append("file":{"type":"string","format":"binary"})`

or maybe a decorator like @formDataWithFiles or @multipart

**Comment by ad-on-is** [comment]
There is already a format decorator for the model field

```ts
  @column()
  // @format(binary)
  declare file: string
```

**Comment by zecar** [comment]
My model does not have a property for that

```
@column()
declare imageUrl: string
```

the endpoint request takes a "file" property, not imageUrl that is a file

**Comment by ad-on-is** [comment]
based on your example, I'd assume your model has other properties too. i.e. name, email, etc... so you'd need to link that file to some property, wouldn't you?

in other words, how do you expect to handle the submitted formdata otherwise?

**Comment by zecar** [comment]
Yes, I have multiple properties

In my particular case, I take the file from the "file" property on the request, alongside other Model properties.

I upload the file and populate Model.imageUrl with the uploaded file's path, then save

**Comment by ad-on-is** [comment]
Well, there you have your answer. then just add a "virtual" field _file_ to your model... doesn't have to be mapped to a column.

**Comment by zecar** [comment]
yes, but then I would have to exclude it from everywhere I have the Model referenced, right?

**Comment by zecar** [comment]
append() does not work and neither does exclude()

it just displays the whole model inside <Model>

**Comment by ad-on-is** [comment]
Fixed in 3.11.0

```
<User>.exclude(id).append("picture":{"type":"string","format":"binary"})
```
